profile/README.md

```diff
@@ -10,3 +10,33 @@
 🍿 Fun facts - what does your team eat for breakfast?
 🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
 -->
+
+
```

Comment:
For resources, I think it would be helpful to have links to our popular public R packages. Maybe start with BPG and VennDiagram? (and add more if we have enough space?)

Reply:
I put both of those and also stats so far.

Reply:
Cool. `BoutrosLab.statistics.general` is not on CRAN yet so maybe remove from the list for now?

Reply:
VennDiagram and BPG publications

https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1281-5
https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2610-2